inference-engine/tests_deprecated/functional/vpu/common/myriad_hw_conv_tests.hpp

```diff
@@ -145,6 +147,18 @@ TEST_P(MyriadX_HW_Convolution_Tests_nightly, WithClamp) {
         SKIP() << "Non-MyriadX device";
     }
 
+    // TODO: Issue: 34466
+    if (CheckMA2085()) {
+        const ::testing::TestInfo* const test_info =
+              ::testing::UnitTest::GetInstance()->current_test_info();
+        const char* test_name = test_info->name();
+        const char* test_case = test_info->test_case_name();
+        if (!strcmp(test_name, "WithClamp/0") &&
+            !strcmp(test_case, "conv_3x3s1p1_vgg/MyriadX_HW_Convolution_Tests_nightly")) {
```

Comment:
Is there a way to check variables values (test parameters) instead of parsing test name + test case?

Reply:
I don't know... What is wrong with parsing test name? This patch excludes the test named as _conv_3x3s1p1_vgg/MyriadX_HW_Convolution_Tests_nightly.WithClamp/0_, which fails according to the issue #-34466.

Reply:
Parsing test name + test case doesn't seem robust to me. If test name changed we will not notice it and just run test, which will probably make us to do another change. If we change variable values we have compile-time safety - if variables names changed and check is not changed it will not compile.

Also, it will look a bit simpler I guess

Reply:
Why should test name change? Isn't it hard-coded at the _hyriad_hw_conv_tests.hpp:463_?

Reply:
Yea, but you trying to disable test only with specific parameters, right? So why just don't check them?

Reply:
Anyway, I think it doesn't worth discussion

Reply:
My patch intends to disable the test named _conv_3x3s1p1_vgg/MyriadX_HW_Convolution_Tests_nightly.WithClamp/0_ on _ma2085_, which fails according to the issue #-34466